src/components/PersonaCard/PersonaCard.Square.html

```diff
@@ -15,31 +15,39 @@
     </div>
   </div>
   <ul class="ms-PersonaCard-actions">
-    <li id="chat" class="ms-PersonaCard-action is-active"><i class="ms-Icon ms-Icon--chat"></i></li>
-    <li id="phone" class="ms-PersonaCard-action"><i class="ms-Icon ms-Icon--phone"></i></li>
-    <li id="video" class="ms-PersonaCard-action"><i class="ms-Icon ms-Icon--video"></i></li>
-    <li id="mail" class="ms-PersonaCard-action"><i class="ms-Icon ms-Icon--mail"></i></li>
-    <li class="ms-PersonaCard-overflow" alt="View profile in Delve" title="View profile in Delve">View profile</li>
-    <li id="org" class="ms-PersonaCard-action ms-PersonaCard-orgChart"><i class="ms-Icon ms-Icon--org"></i></li>
+    <li id="phoneAction" class="ms-PersonaCard-action" data-detailsTargetId="detail-2b"><i class="ms-Icon ms-Icon--phone"></i></li>
+    <li id="chatAction" class="ms-PersonaCard-action" data-detailsTargetId="detail-1b"><i class="ms-Icon ms-Icon--chat"></i></li>
+    <li id="videoAction" class="ms-PersonaCard-action" data-detailsTargetId="detail-3b"><i class="ms-Icon ms-Icon--video"></i></li>
+    <li id="mailAction" class="ms-PersonaCard-action is-active" data-detailsTargetId="detail-4b"><i class="ms-Icon ms-Icon--mail"></i></li>
+    <li id="profileAction" class="ms-PersonaCard-overflow" data-detailsTargetId="detail-5b" data-actionUrl="http://dev.office.com/fabric" alt="View profile in Delve" title="View profile in Delve">View profile</li>
+    <li id="orgAction" class="ms-PersonaCard-action ms-PersonaCard-orgChart" data-detailsTargetId="detail-6b"><i class="ms-Icon ms-Icon--org"></i></li>
   </ul>
   <div class="ms-PersonaCard-actionDetailBox">
-    <ul id="detailList" class="ms-PersonaCard-detailChat">
-      <li id="chat" class="ms-PersonaCard-actionDetails detail-1">
+    <ul id="detail-1b" class="ms-PersonaCard-details">
+      <li  class="ms-PersonaCard-actionDetails">
         <div class="ms-PersonaCard-detailLine"><span class="ms-PersonaCard-detailLabel">Lync:</span> <a class="ms-Link" href="#">Start Lync call</a></div>
       </li>
-      <li id="phone" class="ms-PersonaCard-actionDetails detail-2">
+   </ul>
+   <ul id="detail-2b" class="ms-PersonaCard-details">
+      <li class="ms-PersonaCard-actionDetails">
         <div class="ms-PersonaCard-detailLine"><span class="ms-PersonaCard-detailLabel">Personal:</span> 555.206.2443</div>
         <div class="ms-PersonaCard-detailLine"><span class="ms-PersonaCard-detailLabel">Work:</span> 555.929.8240</div>
       </li>
-      <li id="video" class="ms-PersonaCard-actionDetails detail-3">
+   </ul>
+   <ul id="detail-3b" class="ms-PersonaCard-details">
+      <li class="ms-PersonaCard-actionDetails">
         <div class="ms-PersonaCard-detailLine"><span class="ms-PersonaCard-detailLabel">Skype:</span> <a class="ms-Link" href="#">Start Skype call</a></div>
       </li>
-      <li id="mail" class="ms-PersonaCard-actionDetails detail-4">
+   </ul>
+   <ul id="detail-4b" class="ms-PersonaCard-details">
```

Comment:
Why the change the `detail-4b` instead of `detail-4` here? Is it to prevent conflicts when two PersonaCards are on the same page? If so, we should find a more foolproof way to avoid conflicts.